tests/test_pillow.py

```diff
@@ -50,15 +50,17 @@ def test_write_single_frame(image_files: Path, im_npy: str, im_out: str, im_comp
         # ("newtonscradle_rgba.npy", "iio.gif", "pil.gif"),
     ],
 )
-def test_write_multiframe(image_files: Path, im_npy: str, im_out: str, im_comp: str):
+@pytest.mark.needs_internet
+def test_write_multiframe(image_cache, tmp_path, im_npy, im_out, im_comp):
```

Comment:
@anjos I noticed this in other places within this file, too, but I will only ask here: How come you ended up removing the type annotations from the tests?

Reply:
Frankly, I'm not sure anymore on this one.  Let me check again.